benchcab/benchcab.py

```diff
@@ -163,7 +163,8 @@ def fluxsite_submit_job(self) -> None:
     def checkout(self):
         """Endpoint for `benchcab checkout`."""
 
-        setup_src_dir()
+        with chdir(self.root_dir):
+            mkdir(internal.SRC_DIR, exist_ok=True, verbose=True)
```

Comment:
We can probably remove the `chdir` here? I notice you have used `chdir` elsewhere in the code. Was it required to make the tests pass?
```suggestion
        mkdir(internal.SRC_DIR, exist_ok=True, verbose=True)
```

Reply:
In the current instantiation of the tests, without `mkdir` the paths will be created where the tests are run and not in the temporary location. The paths are deleted right away in the tests so nobody will see them but it is still an undesired side effect. Especially if something goes wrong and they are not deleted.

With the redesign of the tests, since we have a fixture that changes the CWD for the whole test, it will work without `chdir`.

I'm happy to wait for the tests redesign to go in first.

Reply:
I'm surprised the files are not being created in the temporary location since there is a `run_around_tests()` autouse fixture which changes the CWD. But I'm happy for this to wait until #172 goes in.

Reply:
I'll double check...